tests/results/24a70216d2914871c4c1f0f4a7aa5f8fc1c6d0e1b2c2718f62ade6903680aaeb_slashdot/kangooroo/results.json

```diff
@@ -1,33 +1,61 @@
-    "strict-transport-security": "max-age\u003d31536000",
-    "CF-Cache-Status": "DYNAMIC",
-    "Server": "cloudflare",
-    "CF-RAY": "81c471759e64ab3b-YYZ",
-    "Content-Encoding": "br",
-    "alt-svc": "h3\u003d\":443\"; ma\u003d86400"
+  "experiment": {
+    "engineInfo": {
+      "engineName": "Standalone-Kangooroo",
+      "engineVersion": "2.0.1"
+    },
+    "params": {
+      "url": "https://news.slashdot.org/comments.pl?sid=11253499&cid=55399795",
+      "urlType": "PHISHING",
+      "windowSize": "1280x720",
+      "userAgent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/80.0.3987.163 Safari/537.36",
+      "modules": [
+        "summary"
+      ]
+    },
+    "execution": {
+      "status": "SUCCESS",
+      "messages": [],
+      "startTime": "Mon Nov 25 18:56:18 UTC 2024",
+      "processTime": 55710
+    }
   },
-  "has_timed_out": false,
-  "processing_time": 30477,
-  "messages": [],
-  "creationDate": "Oct 26, 2023, 5:33:43 PM",
-  "engineName": "Standalone-Kangooroo",
-  "engineVersion": "1.0.1"
-}
\ No newline at end of file
+  "summary": {
+    "fetchResult": {
+      "response_code": 200,
+      "connection_success": true,
```

Comment:
Are those three boolean supposed to be actual booleans or strings? They are stored as strings in other results.json files.